components/homme/src/theta-l_kokkos/cxx/ForcingFunctor.hpp

```diff
@@ -99,8 +99,7 @@ class ForcingFunctor
     m_hydrostatic = p.theta_hydrostatic_mode;
     m_qsize = p.qsize;
 
-    // TODO: this may change, depending on the simulation params
-    m_adjust_ps = true;
+    m_adjust_ps = (p.dt_remap_factor == 0);
```

Comment:
If `p.dt_remap_factor == 0`, preserve old behavior.

Reply:
by some reason i thought scream is already using adjust_dp3d, but looking at the code it does not. When this goes to scream, it will cause nonbfb.

Reply:
Yes, every test with homme against baselines should fail. I'm going to create a PR when this is merged and look at the diffs.